server.go

```diff
@@ -416,6 +430,18 @@ func (s *Server) Start() {
 		},
 	}
 
+	for _, sink := range s.spanSinks {
+		if err := sink.Start(s.TraceClient); err != nil {
```

Comment:
I'd love to see an info level `Starting sink.Name()` here, as it's quite helpful to see this when inpsecting a running veneur and adds very little log overhead as it's a singular line.